Makefile

```diff
@@ -139,78 +131,69 @@ test-dependencies:
 	sudo chmod +x $(KUBEBUILDER_DIR)/kubebuilder
 	curl -L "https://go.kubebuilder.io/test-tools/$(K8S_VERSION)/$(GOOS)/$(GOARCH)" | sudo tar xz --strip-components=2 -C $(KUBEBUILDER_DIR)/
 
+GOSEC ?= $(GOBIN)/gosec
 $(GOSEC):
-	curl -L https://github.com/securego/gosec/releases/download/v$(GOSEC_VERSION)/gosec_$(GOSEC_VERSION)_$(GOOS)_$(GOARCH).tar.gz | tar -xz -C /tmp/
-	sudo mv /tmp/gosec $(GOSEC)
+	go install github.com/securego/gosec/v2/cmd/gosec@v2.9.6
 
+.PHONY: gosec-scan
 gosec-scan: $(GOSEC)
 	$(GOSEC) -fmt sonarqube -out gosec.json -no-fail -exclude-dir=.go ./...
 
 ############################################################
 # build section
 ############################################################
 
+.PHONY: build
 build:
 	@build/common/scripts/gobuild.sh build/_output/bin/$(IMG) ./
 
+.PHONY: local
 local:
 	@GOOS=darwin build/common/scripts/gobuild.sh build/_output/bin/$(IMG) ./
 
+.PHONY: run
 run:
 	HUB_CONFIG=$(HUB_CONFIG) MANAGED_CONFIG=$(MANAGED_CONFIG) WATCH_NAMESPACE=$(WATCH_NAMESPACE) go run ./main.go --leader-elect=false
 
 ############################################################
 # images section
 ############################################################
 
+.PHONY: build-images
 build-images:
 	@docker build -t ${IMAGE_NAME_AND_VERSION} -f build/Dockerfile .
 	@docker tag ${IMAGE_NAME_AND_VERSION} $(REGISTRY)/$(IMG):$(TAG)
 
 ############################################################
 # clean section
 ############################################################
+.PHONY: clean
 clean::
 	rm -f build/_output/bin/$(IMG)
```

Comment:
Unless I'm mistaken, I think this should also include removing installed binaries from `bin/` (since they also won't be replaced/upgraded unless they are removed).

Reply:
That's a good point. We will need to address that, I think it would be good to fit in when the `go get` commands actually are removed (I pulled those changes out of this PR)